src/main/java/org/spongepowered/api/data/property/store/PropertyStore.java

```diff
@@ -81,11 +74,13 @@
      * <p>The priority is a measurement where the higher the priority, the
      * sooner this {@link PropertyStore} is queried such that if
      * {@link #getFor(PropertyHolder)} is called and returns a present value,
-     * that present value is returned. Usually, Sponge implemented
-     * {@link PropertyStore}s have a priority of <code>100</code>.</p>
+     * that present value is returned. The default priority of a
+     * {@link PropertyStore} is <code>100</code>.</p>
      *
      * @return The priority
      */
-    int getPriority();
+    default int getPriority() {
+        return 100;
+    }
 
```

Comment:
May want to just have a `getProperty` for easy getter verification that you're using the right store.

Reply:
How will we handle directional properties without the other method? Or do you just mean renaming `getFor` to `getProperty`?

Reply:
No, as in get property type method returning Property, not the value of the property.

Reply:
You don't need to know the `Property` (Type) of a store? You register a store for a specific `Property`.
https://github.com/SpongePowered/SpongeAPI/pull/1871/files#diff-f470a88385428e41ad4e5cb915ee3982R44